<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSD</class>
 <widget class="QDialog" name="OSD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="2">
    <widget class="QPushButton" name="pushCancel">
     <property name="toolTip">
      <string>Cancel</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="style.qrc">
       <normaloff>:/rc/dialog-cancel.png</normaloff>:/rc/dialog-cancel.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QProgressBar" name="progressAudio">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QProgressBar" name="progressVideo">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelAudio">
     <property name="text">
      <string>Audio</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelVideo">
     <property name="text">
      <string>Video</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
